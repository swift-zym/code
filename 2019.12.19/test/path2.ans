Yes
1 10 18 11 17 6 19 12 3 2 9 16 20 4 8 14 5 13 7 15 
