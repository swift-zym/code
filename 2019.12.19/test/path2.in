20
8 1
8 10
8 6
10 11
1 7
6 19
10 17
8 20
6 12
6 16
7 14
16 3
1 15
16 2
14 5
16 9
11 18
14 13
8 4
