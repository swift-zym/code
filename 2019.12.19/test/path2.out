Yes
1 8 10 11 18 17 4 7 5 13 14 15 6 19 2 16 3 9 12 20
